### ISSUE TITLE
How to use this plugin for deploy?

### ISSUE THREAD

**Issue by MarcDahlem** [johndevs/gradle-vaadin-plugin#475]
Hi Johndevs,

is there a possibility to use this plugin to bundle a deploy unit?
E.g. is there a possibility to get a working war file or fat jar file containing all files needed?

Thanks & br

**Comment by johndevs** [comment]
Hi, yes that is fully possible! 

To create a WAR archive you simply run the war task and it will create a the archive under build/libs. You can deploy that on any server.

To create a fat jar you will need to either create your own task that includes yhe dependencies nd create a manifest or simply rely on spring boot for example to package a fat jar for you.

**Comment by quayal** [comment]
Hello John,

I ran gradlew war task with your plugin (couldn't find anything like war in the vaadin task folder), and the resulting war had empty WEB-INF/lib folder. When I copied all the dependency jars from payara folder, it worked for the exploded war, but the compressed stil complained about classes missing (precisely java.lang.ClassNotFoundException: com.vaadin.server.VaadinServlet)

Am I missing something?

**Comment by johndevs** [comment]
Hi, running the war task as you did should produce a working archive with the dependencies included. Could you post the build.gradle you are using so I could try reproducing the issue?

**Comment by quayal** [comment]
Hi,
Here are the steps I did:
1. Deleted everything from build/libs folder to give gradle another chance to do its job
2. Ran gradlew war (gradle version 4.6, but with 4.0 installed previously it didn't work either)
3. war's WEB-INF only contains classes folder, no lib folder, and obviously no dependency jars inside.

This is just my learning project, so the build.gradle is basic, but I'm also taking this opportunity to learn how to deploy to an independent server. The war built with Maven works and can be deplyed to Tomcat. Here's build.grade:
`plugins {
    id 'com.devsoap.plugin.vaadin' version '1.3.1'
}

group 'com.quayal'
version '1.0-SNAPSHOT'

apply plugin: 'java'

sourceCompatibility = 1.8

repositories {
    mavenCentral()
}

dependencies {
    testCompile group: 'junit', name: 'junit', version: '4.12'
    }
`
